gong_setup.sh

```diff
@@ -29,25 +29,32 @@ function changeProjectName() {
   new_path=$(sed "s/\./\//g" <<<"$PACKAGE_NAME")
   gong_new_path=$(sed "s/\./\//g" <<<"$BASE_PROJECT_PAKAGE_NAME")
   first_gong_folder=$(sed 's/\..*//' <<<"$BASE_PROJECT_PAKAGE_NAME")
-  cd "app/src/" || exit 1
+
+  cd "app/" || exit 1
+  sed -i "s/$BASE_PROJECT_PAKAGE_NAME/$PACKAGE_NAME/" proguard-rules.pro
+  sed -i "s/$BASE_PROJECT_NAME/$REAL_PROJECT_NAME/" proguard-rules.pro
+
+  cd "src/" || exit 1
 
   if [ -d "$TEMPORAL_FOLDER" ]; then rm -Rf $TEMPORAL_FOLDER; fi
   mkdir "$TEMPORAL_FOLDER"
+  
+  for entry in $fileNames 
+  do
```

Comment:
i'd join it to be consistent with the if sentences
```suggestion
  for entry in $fileNames ;  do
```

Reply:
where `fileNames` is defined?

Reply:
weird i thought i do it different